.github/workflows/dependencies.yml

```
@@ -0,0 +1,46 @@
+      - name: Update dependencies
+        run: |
+          # Create requirements.in if it doesn't exist
+          if [ ! -f requirements.in ]; then
+            cp requirements.txt requirements.in
+          fi
+
+          # Update dependencies
+          pip-compile --upgrade requirements.in
+
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v5
```

actionlint (1.7.4)

the runner of "peter-evans/create-pull-request@v5" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)